libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/Flyout.tsx

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  Icon,
+  Image,
+  ImageFit,
+  List,
+  Panel,
+  PanelType,
+  FocusZone,
+  Stack,
+  Text,
+  Separator
+} from "@fluentui/react";
+import { localization } from "@responsible-ai/localization";
+import React from "react";
+
+import { IDatasetSummary } from "../Interfaces/IExplanationDashboardProps";
+
+import { flyoutStyles } from "./Flyout.styles";
+import { IListItem } from "./ImageList";
+
+export interface IFlyoutProps {
+  data: IDatasetSummary;
+  isOpen: boolean;
+  item: IListItem | undefined;
+  callback: () => void;
+}
+
+export class IFlyoutState {}
+
+const stackTokens = {
+  childrenGap: "l1"
+};
+
+export class Flyout extends React.Component<IFlyoutProps, IFlyoutState> {
+  public constructor(props: IFlyoutProps) {
+    super(props);
+    this.state = {};
+  }
+
+  public render(): React.ReactNode {
+    const classNames = flyoutStyles();
+
+    const item = this.props.item;
```

Comment:
nit:
`const { data, isOpen, item, callback } = this.props`. So we can directly use the constant without this.props. for all code below